game/stats.cc

```diff
@@ -0,0 +1,33 @@
+#include "stats.h"
+
+using namespace std;
+
+const int Stats::getStat(StatName s) {
+  switch(s) {
+    case HP:
+      return hp;
+    case ATTACK:
+      return attack;
+    case DEFENSE:
+      return defense;
+    case SPEED:
+      return speed;
+  }
+}
+
+void Stats::setStat(StatName s, int newValue) {
+  switch(s) {
+    case HP:
+      hp = newValue;
+      break;
+    case ATTACK:
+      attack = newValue;
```

Comment:
ATTACK, DEFENSE, and SPEED should check to make sure they are within their bounds. I think we agreed on +/- 5? Maybe you could make this a static variable in the `Stats` class